packages/i18n-calypso-cli/.eslintrc.js

```diff
@@ -6,4 +6,12 @@ module.exports = {
 		'import/no-nodejs-modules': 0,
 		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
 	},
+	overrides: [
+		{
+			files: [ '*.md.js' ],
```

Comment:
`*.md.js` was intentional, right? We're targeting JS blocks inside markdown files?

Reply:
Yes. The plugin we are using uses this format to specify "codeblocks of language 'js' inside markdown" (https://www.npmjs.com/package/eslint-plugin-md#modifying-eslint-setup-for-js-code-inside-md-files)

The architecture is actually quite clever: the plugin uses an eslint preprocessor to receive a single Markdown file as input and produces one Markdown without code blocks + n files named 'file.md.<lang>' for each code block. Then lints each one of those individually, using the regular JavaScript rules to lint `file.md.js`. And then aggregates all the errors from each "sub-file" tweaking line and column numbers so it makes sense as a whole.